doc/release/1.15.0-notes.rst

```diff
@@ -125,22 +147,24 @@ types from ``ctypes``.
 
 ``np.ma.notmasked_contiguous`` and ``np.ma.flatnotmasked_contiguous`` always return lists
 -----------------------------------------------------------------------------------------
-This was always the documented behavior, but in reality the result used to be
-any of slice, None, or list.
-
-All downstream users seem to use detect the `None` result from
-``flatnotmasked_contiguous`` and replace it with ``[]``.
-These callers will continue to work as before.
-
-``np.squeeze`` now respects the API expectation of objects that do not handle an ``axis`` argument
---------------------------------------------------------------------------------------------------
-Prior to version ``1.7.0`` ``np.squeeze`` did not have an ``axis`` argument and all empty axes were removed
-by default. After incorporation of an ``axis`` argument, it was possible to selectively squeeze single
-or multiple empty axes, but the old API expectation was not respected because the axes could still be
-selectively removed (silent success) in an object depending on the old API. The silent success is no
-longer possible, and objects expecting the old API are respected. The silent success was prevented
-by removing the interception of an otherwise-normal Exception when ``axis`` was provided to an object
-using the old API.
+This is the documented behavior, but previously the result could be any of
+slice, None, or list.
+
+All downstream users seem to check for the `None` result from
+``flatnotmasked_contiguous`` and replace it with ``[]``.  Those callers will
+continue to work as before.
+
+``np.squeeze`` API expectation of objects that cannot handle an ``axis`` argument
```

Comment:
This doesn't feel grammatically meaningful any more

Reply:
I find the whole entry confusing and have attempted a rewrite.
```
``np.squeeze`` restores old behavior of objects that cannot handle an ``axis`` argument
---------------------------------------------------------------------------------------
Prior to version ``1.7.0``, `np.squeeze` did not have an ``axis`` argument and
all empty axes were removed by default. The incorporation of an ``axis``
argument made it possible to selectively squeeze single or multiple empty axes,
but the old API expectation was not respected because axes could still be
selectively removed (silent success) from an object expecting all empty axes to
be removed. That silent, selective removal of empty axes for objects expecting
the old behavior has been fixed and the old behavior restored.
```
Better?